main/lsp/LSPTypechecker.cc

```diff
@@ -263,14 +264,20 @@ vector<core::FileRef> LSPTypechecker::runFastPath(LSPFileUpdates &updates, Worke
                 const auto &oldMethodHashes = oldSymbolHashes.methodHashes;
                 const auto &newMethodHashes = newSymbolHashes.methodHashes;
 
+                // TODO(jez) is this critical for the set_difference to work, or just a sanity check?
```

Comment:
This is not critical for `set_difference` to work (`set_difference` just needs sorted sets), but it does make sense in the previous world where the method hashes for each file should have been the same.